### PR TITLE
Move model loading to the correct event. Fixes #2991

### LINKED ISSUE #2991: Missing Texturs with newerst Forge [1.12]
I'm just gonna relink the bug report because it looks like it is exactly the same.

https://github.com/thiakil/Applied-Energistics-2/issues/38

### PR DESCRIPTION
**Pull request by fscan**
Also rename method to a sane name.

Fixes #2991

### CODE REVIEW

**Comment by orod-org** [comment]
SonarQube analysis reported 32 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 4 critical
* ![MAJOR][MAJOR] 13 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 10 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [ModelRegComponent.java](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/bootstrap/components/ModelRegComponent.java): Add or update the header of this file. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1451)
1. ![CRITICAL][CRITICAL] [Registration.java#L153](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L153): Define a constant instead of duplicating this literal "Past the registration phase already!" 3 times. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [Registration.java#L390](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L390): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L403](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L403): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L456](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L456): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [Registration.java#L183](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L183): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L214](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L214): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L220](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L220): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L223](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L223): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L225](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/778faba37b2e0f414dd4f8df2a1e0d5c5fbd4f40/src/main/java/appeng/core/Registration.java#L225): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by yueh** [comment]
As a more general feedbacks, it is more about the bootrap and not the model registration in particular.

The first issue I currently see is that Forge will add more events. So it will certainly become a bit crowded with a specific handler method for every possible event. With `preInit`, `init`, and `postInit` it was still fine. But now we also have the model registry, block and what not.
Therefore I would consider moving from `FeatureFactory#getBootstrapComponents()` to a more specific `FeatureFactory#getComponents(? extends IBootstrapComponent.class)`, so not every `IBootstrapComponent` has to implement an empty method for useless events.

It would also avoid something like `ItemModelComponent implements IBootstrapComponent`, which should be `ItemModelComponent implements ModelRegistrationComponent`. `IBootstrapComponent` might look like the right option at first glance with all methods being default methods. (To solve the previous issue). But `ModelRegistrationComponent` makes `modelRegistration()` explicitly a interface method again, to avoid someone forgetting to implement it with a concrete component.

A change could probably then also swap the class hierarchy. E.g. something like `ConcreteComponent implements IInitComponent, IModelComponent, IBlockComponent`. `IBootstrapComponent` might just be kept as common super interface to indicate it can be used to register a new component type to the featyre factory.

This is also an idea to get rid of `Registration.java` and handle everything in `appeng.bootstrap`
But that is just a abstract idea about `appeng.bootstrap`, not this particular PR.

A actual issue I have with this, is the static call to `ModelLoader`, I know it is the way forge wants it. But it is still not ideal to have dozens of static classes around, which are only valid for a specific event. They simply should be available through the event object.
So at least `IBootstrapComponent#modelRegistration( Side side )` should be `IBootstrapComponent#modelRegistration( Side side, ModelLoader modelLoader )` and we pass the static class to it. Should Forge ever change it, we do not have to touch everything calling the static class directly.
This of course would apply to any other component type.

Also the previously mentioned `implements IBootstrapComponent` not `ModelRegComponent`

**Comment by orod-org** [comment]
SonarQube analysis reported 32 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 4 critical
* ![MAJOR][MAJOR] 13 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 10 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [ModelRegComponent.java](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/bootstrap/components/ModelRegComponent.java): Add or update the header of this file. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1451)
1. ![CRITICAL][CRITICAL] [Registration.java#L153](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L153): Define a constant instead of duplicating this literal "Past the registration phase already!" 3 times. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [Registration.java#L390](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L390): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L403](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L403): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L456](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L456): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [Registration.java#L183](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L183): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L214](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L214): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L220](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L220): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L223](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L223): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L225](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/8b10c1977f4af7bbe8db04abd05841fb5ada5f47/src/main/java/appeng/core/Registration.java#L225): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by fscan** [comment]
As i see it, the  `I<stage>Component` interfaces are just to allow passing a quick lambda to the registry. Extending the `IBoostrapComponent` interface at a later stage should not impact them because of the default implementations.

I'm not sure how the `ModelLoader` parameter would work. We call only static methods, we don't have access to an instance of `ModelLoader` at any stage.

File: src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java [hunk 1]
```diff
@@ -36,10 +36,11 @@
...
 
+import appeng.bootstrap.IBootstrapComponent;
 import appeng.core.AppEng;
 
 
-public class ModelOverrideComponent implements PreInitComponent
+public class ModelOverrideComponent implements IBootstrapComponent
 {
 
 	private static final ModelResourceLocation MODEL_MISSING = new ModelResourceLocation( "builtin/missing", "missing" );
```
Comment (orod-org) [inline src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java]: MAJOR Remove this unused "MODEL_MISSING" private field. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)

**Comment by orod-org** [comment]
SonarQube analysis reported 35 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 5 critical
* ![MAJOR][MAJOR] 15 major
* ![MINOR][MINOR] 4 minor
* ![INFO][INFO] 10 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [TileEntityComponent.java](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/bootstrap/components/TileEntityComponent.java): Add or update the header of this file. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1451)
1. ![CRITICAL][CRITICAL] [TileEntityComponent.java#L23](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/bootstrap/components/TileEntityComponent.java#L23): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [Registration.java#L153](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L153): Define a constant instead of duplicating this literal "Past the registration phase already!" 3 times. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [Registration.java#L390](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L390): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L403](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L403): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L456](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L456): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [ModelOverrideComponent.java#L67](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java#L67): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L183](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L183): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L214](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L214): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L220](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/656254e9f31c51ebfe59e5676ddf81426e1e9cdd/src/main/java/appeng/core/Registration.java#L220): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by GuntherDW** [comment]
While most of those registration events still have "// TODO : 1.12 Improve" comments, they should get moved around or even get a new class of their own to handle the Forge Registry event.

Right now it just kind of looks ugly because it is just shuffled around code in Registration.java to make it work with those events. But this is why I labeled them with "1.12 Improve".
With their previous iteration I would agree that "ModelLoader" does not really fit in that callback class, but as time goes on Forge will probably add more events, and especially now in 1.12 they will get more strict in terms of which stage to load your stuff and inject it back in to Forge in.

File: src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java [hunk 2]
```diff
@@ -39,7 +39,7 @@
 import appeng.core.AppEng;
 
 
-public class ModelOverrideComponent implements PreInitComponent
+public class ModelOverrideComponent implements IPreInitComponent
 {
 
 	private static final ModelResourceLocation MODEL_MISSING = new ModelResourceLocation( "builtin/missing", "missing" );
```
Comment (orod-org) [inline src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java]: MAJOR Remove this unused "MODEL_MISSING" private field. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)

**Comment by orod-org** [comment]
SonarQube analysis reported 37 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![CRITICAL][CRITICAL] 6 critical
* ![MAJOR][MAJOR] 15 major
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 10 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![BLOCKER][BLOCKER] [TileEntityComponent.java](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/bootstrap/components/TileEntityComponent.java): Add or update the header of this file. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1451)
1. ![CRITICAL][CRITICAL] [BlockDefinitionBuilder.java#L205](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/bootstrap/BlockDefinitionBuilder.java#L205): The Cyclomatic Complexity of this method "build" is 11 which is greater than 10 authorized. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
1. ![CRITICAL][CRITICAL] [TileEntityComponent.java#L22](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/bootstrap/components/TileEntityComponent.java#L22): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [Registration.java#L158](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L158): Define a constant instead of duplicating this literal "Past the registration phase already!" 3 times. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [Registration.java#L396](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L396): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L409](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L409): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![CRITICAL][CRITICAL] [Registration.java#L462](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L462): Make the enclosing method "static" or remove this set. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)
1. ![MAJOR][MAJOR] [ModelOverrideComponent.java#L66](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/bootstrap/components/ModelOverrideComponent.java#L66): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L188](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L188): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
1. ![MAJOR][MAJOR] [Registration.java#L219](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/4ccd7781dd84a0cc3eb0dc9c3257de741eaff574/src/main/java/appeng/core/Registration.java#L219): This block of commented-out lines of code should be removed. [rule](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**